<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Window</class>
 <widget class="QMainWindow" name="Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>404</width>
    <height>370</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Window</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontal_connection">
      <item>
       <widget class="QLabel" name="_device">
        <property name="text">
         <string>Device</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="combo_device">
        <item>
         <property name="text">
          <string>/dev/ttyUSB0</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>/dev/ttyUSB1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>/dev/ttyUSB2</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="_baud">
        <property name="text">
         <string>Baud</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="combo_baud">
        <item>
         <property name="text">
          <string>9600</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>11500</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button_connect">
        <property name="text">
         <string>Connect</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QFrame" name="frame_controls">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="_analog_read">
         <property name="text">
          <string>analogRead</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="0" rowspan="2">
        <layout class="QGridLayout" name="grid_analog_read">
         <item row="5" column="0">
          <widget class="QLabel" name="_analog_5">
           <property name="text">
            <string>A5</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="_analog_4">
           <property name="text">
            <string>A4</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QSpinBox" name="analog_4">
           <property name="readOnly">
            <bool>true</bool>
           </property>
           <property name="maximum">
            <number>1023</number>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QSpinBox" name="analog_3">
           <property name="readOnly">
            <bool>true</bool>
           </property>
           <property name="maximum">
            <number>1023</number>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QSpinBox" name="analog_2">
           <property name="readOnly">
            <bool>true</bool>
           </property>
           <property name="maximum">
            <number>1023</number>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="_analog_3">
           <property name="text">
            <string>A3</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QSpinBox" name="analog_5">
           <property name="readOnly">
            <bool>true</bool>
           </property>
           <property name="maximum">
            <number>1023</number>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QSpinBox" name="analog_1">
           <property name="readOnly">
            <bool>true</bool>
           </property>
           <property name="maximum">
            <number>1023</number>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="_analog_1">
           <property name="text">
            <string>A1</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="_analog_2">
           <property name="text">
            <string>A2</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="_analog_0">
           <property name="text">
            <string>A0</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QSpinBox" name="analog_0">
           <property name="readOnly">
            <bool>true</bool>
           </property>
           <property name="maximum">
            <number>1023</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="2" rowspan="2">
        <layout class="QGridLayout" name="grid_digital_write">
         <item row="0" column="0">
          <widget class="QCheckBox" name="check_digital_0">
           <property name="text">
            <string>D0</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QCheckBox" name="check_digital_8">
           <property name="text">
            <string>D8</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QCheckBox" name="check_digital_5">
           <property name="text">
            <string>D5</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QCheckBox" name="check_digital_1">
           <property name="text">
            <string>D1</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QCheckBox" name="check_digital_12">
           <property name="text">
            <string>D12</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QCheckBox" name="check_digital_6">
           <property name="text">
            <string>D6</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QCheckBox" name="check_digital_13">
           <property name="text">
            <string>D13</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QCheckBox" name="check_digital_7">
           <property name="text">
            <string>D7</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QCheckBox" name="check_digital_2">
           <property name="text">
            <string>D2</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QCheckBox" name="check_digital_9">
           <property name="text">
            <string>D9</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QCheckBox" name="check_digital_3">
           <property name="text">
            <string>D3</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QCheckBox" name="check_digital_10">
           <property name="text">
            <string>D10</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QCheckBox" name="check_digital_4">
           <property name="text">
            <string>D4</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QCheckBox" name="check_digital_11">
           <property name="text">
            <string>D11</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="_analog_write">
         <property name="text">
          <string>analogWrite</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="1" rowspan="2">
        <layout class="QGridLayout" name="grid_analog_write">
         <item row="5" column="0">
          <widget class="QLabel" name="_pwm_11">
           <property name="text">
            <string>~11</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QSlider" name="pwm_11">
           <property name="maximum">
            <number>255</number>
           </property>
           <property name="singleStep">
            <number>32</number>
           </property>
           <property name="pageStep">
            <number>32</number>
           </property>
           <property name="tracking">
            <bool>false</bool>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="5" column="2">
          <widget class="QLabel" name="value_11">
           <property name="minimumSize">
            <size>
             <width>30</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="QLabel" name="value_9">
           <property name="minimumSize">
            <size>
             <width>30</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QLabel" name="value_6">
           <property name="minimumSize">
            <size>
             <width>30</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="_pwm_9">
           <property name="text">
            <string>~9</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QSlider" name="pwm_10">
           <property name="maximum">
            <number>255</number>
           </property>
           <property name="singleStep">
            <number>32</number>
           </property>
           <property name="pageStep">
            <number>32</number>
           </property>
           <property name="tracking">
            <bool>false</bool>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="4" column="2">
          <widget class="QLabel" name="value_10">
           <property name="minimumSize">
            <size>
             <width>30</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QSlider" name="pwm_3">
           <property name="maximum">
            <number>255</number>
           </property>
           <property name="singleStep">
            <number>32</number>
           </property>
           <property name="pageStep">
            <number>32</number>
           </property>
           <property name="tracking">
            <bool>false</bool>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="_pwm_5">
           <property name="text">
            <string>~5</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="_pwm_10">
           <property name="text">
            <string>~10</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QSlider" name="pwm_9">
           <property name="maximum">
            <number>255</number>
           </property>
           <property name="singleStep">
            <number>32</number>
           </property>
           <property name="pageStep">
            <number>32</number>
           </property>
           <property name="tracking">
            <bool>false</bool>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="value_3">
           <property name="minimumSize">
            <size>
             <width>30</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="_pwm_6">
           <property name="text">
            <string>~6</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QSlider" name="pwm_6">
           <property name="maximum">
            <number>255</number>
           </property>
           <property name="singleStep">
            <number>32</number>
           </property>
           <property name="pageStep">
            <number>32</number>
           </property>
           <property name="tracking">
            <bool>false</bool>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QSlider" name="pwm_5">
           <property name="maximum">
            <number>255</number>
           </property>
           <property name="singleStep">
            <number>32</number>
           </property>
           <property name="pageStep">
            <number>32</number>
           </property>
           <property name="tracking">
            <bool>false</bool>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLabel" name="value_5">
           <property name="minimumSize">
            <size>
             <width>30</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="_pwm_3">
           <property name="text">
            <string>~3</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="_digital_write">
         <property name="text">
          <string>digitalWrite</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>404</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionReset"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionReset">
   <property name="text">
    <string>Reset</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pwm_3</sender>
   <signal>valueChanged(int)</signal>
   <receiver>value_3</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>419</x>
     <y>84</y>
    </hint>
    <hint type="destinationlabel">
     <x>448</x>
     <y>84</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pwm_5</sender>
   <signal>valueChanged(int)</signal>
   <receiver>value_5</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>432</x>
     <y>112</y>
    </hint>
    <hint type="destinationlabel">
     <x>446</x>
     <y>112</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pwm_6</sender>
   <signal>valueChanged(int)</signal>
   <receiver>value_6</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>396</x>
     <y>151</y>
    </hint>
    <hint type="destinationlabel">
     <x>446</x>
     <y>156</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pwm_9</sender>
   <signal>valueChanged(int)</signal>
   <receiver>value_9</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>434</x>
     <y>184</y>
    </hint>
    <hint type="destinationlabel">
     <x>446</x>
     <y>184</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pwm_10</sender>
   <signal>valueChanged(int)</signal>
   <receiver>value_10</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>426</x>
     <y>212</y>
    </hint>
    <hint type="destinationlabel">
     <x>446</x>
     <y>217</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pwm_11</sender>
   <signal>valueChanged(int)</signal>
   <receiver>value_11</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>427</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>446</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
